### ISSUE TITLE
Error: Cannot read property 'ecmaFeatures' of undefined - EspreeFacade.ts:82

### ISSUE THREAD

**Issue by Nantris** [javascript-obfuscator/gulp-javascript-obfuscator#14]
```
C:\projects\myApp\node_modules\javascript-obfuscator\dist\webpack:\JavaScriptObfuscator\src\EspreeFacade.ts:82
            throw new Error(errorMessage);
                  ^
Error: Cannot read property 'ecmaFeatures' of undefined
```

Any thoughts on this? Used to work fine and I can't find anything that's changed.

**Comment by shaji-Dev** [comment]
Mine too, used to work just fine, and all of the sudden this error started to appear... did a clean install on all the packages and still can't find any fix for it.

**Comment by Slapbox** [comment]
@shaji-Dev I don't know if this helps you, but I found the issue did not exist using the plain `javascript-obfuscator` package. It was a slight pain to build a custom file to process and replace my JavaScript with the obfuscated versions, but at least it works.

**Comment by shaji-Dev** [comment]
Yes, I know, it works just fine when you use it directly from the terminal, but I want to use it in programatically in my custom Node.js code and this is where the error occurs... Don't know why the same code works on my other PC but not on this one. It has the same packages and the same versions.
